src/theme/src/components/text-dropdown-button.js

```diff
@@ -0,0 +1,56 @@
+import { minorScale } from '../../../scales'
+import { controlStyles } from '../shared'
+
+export default function getTextDropdownButtonStyles(theme) {
+  const { tokens } = theme
+  const disabledStyles = controlStyles._disabled
+
+  return {
+    baseStyle: {
+      WebkitFontSmoothing: 'antialiased',
+      WebkitAppearance: 'none',
+      MozAppearance: 'none',
+      verticalAlign: 'middle',
+      textDecoration: 'none',
+      border: 'none',
+      fontFamily: tokens.fontFamilies.ui,
+      paddingX: minorScale(1),
+      marginX: minorScale(-1),
+      paddingY: 2,
+      marginY: -2,
+      outline: 'none',
+      cursor: 'pointer',
+      background: 'none',
+      borderRadius: tokens.borderRadius
+    },
+
+    appearances: {
+      default: {
+        _hover: {},
+        _disabled: { ...disabledStyles },
+        _focus: {
+          boxShadow: `0 0 0 2px ${tokens.colors.blue100}`,
```

Comment:
Should this whole value be a theme token   (focus ring)?

Reply:
Yeah - wanted to do that in a more broad change later :thumbsup: